03-Mains/Neutral_density.ipynb

```diff
@@ -6,14 +6,24 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def neutral_density(t, s, indexes):
</pre><p>why not</p><p> </p><p><code></code>`python</p>

<pre class="ql-syntax" spellcheck="false">def neutral_density(t, s, time_index, longitude_index):
</pre><p>That seems much more intuitive and less prone to errors (e.g. giving <code>indexes</code> list with a wrong order). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/627/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>